lib/intercom-rails/proxy/user.rb

```diff
@@ -31,11 +31,11 @@ def self.current_in_context(search_object)
           begin
             user_proxy = new(search_object.instance_eval(&potential_object), search_object)
             return user_proxy if user_proxy.valid?
+            raise ExcludedUserFoundError if user_proxy.excluded?
```

Comment:
we just return user_proxy in this case cause it would never go to this line `raise ExcludedUserFoundError if user_proxy.excluded?`.